sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -596,6 +596,30 @@ static struct snd_soc_dai_link_component dummy_component[] = {
 	}
 };
 
+static int sof_rt5682_check_ssp_port_overwrite(struct device *dev,
+					       struct snd_soc_dai_link_component *dlc)
+{
+	struct snd_soc_dai *dai;
+	struct snd_soc_component *component;
+	int port, ret = -1;
+
+	dai = snd_soc_find_dai(dlc);
+	if (!dai) {
+		dev_err(dev, "fail to get dai\n");
+		return -EINVAL;
+	}
+
+	component = dai->component;
+
+	ret = device_property_read_u32(component->dev, "intel,ssp-port", &port);
+	if (ret) {
+		dev_err(component->dev, "fail to read property 'intel,ssp-port'\n");
+		return -ENODATA;
```

Comment:
how would this work with a kernel update where CoreBoot is not updated? You would return an error then?